cmplibrary/src/androidTest/java/com/sourcepoint/cmplibrary/data/local/DataStorageGdprImplTest.kt

```diff
@@ -13,7 +13,7 @@ class DataStorageGdprImplTest {
     fun clear_data_DataStorage() {
```

Comment:
I think the same should be here, just for the sake of getting more descriptive in the namings of our test cases

Reply:
@carmelo-iriti was this addressed?

Reply:
@andresilveirah we can decide together with @bohdan-go-wombat how to rename all the tests in a way that makes everyone happy